process/OpenHW-Specification-Process.md

```diff
@@ -0,0 +1,152 @@
+| :---		| :------ 									| :-------------------------------------------------------------------------------- |
+| 1    		| License options for contributions: 		|																					|
+| 1.1		| Apache 2.0								| As per OpenHW Member Agreement 													|
+| 1.2		| Solderpad 2.1								| As per OpenHW Member Agreement 													|
+| 1.3		| Commons Attribution Share Alike 4.0 International license | Agreed by OpenHW CEO April 2023									| 
+| 2			| Patent Grant included in contribution license:  | 																				|
+| 2.1		| Apache 2.0								| Yes 																				|
+| 2.2		| Solderpad 2.1								| Yes																				|
+| 2.3		| Commons Attribution Share Alike 4.0 International license | No																| 
+| 3			| Essential patents must be disclosed by contributor | No 																		| 
+| 4			| Contribution Process 						| EDP/ Committers merging pull requests	/ Contributors sign ECA or MCCA				| 			 
+
+
+#### Publication of the Specification
+
+| Number 	| Issue 									| Comment                                               							|
+| :---		| :------ 									| :-------------------------------------------------------------------------------- |
+| 1 		| Specification published by				| OpenHW on Github and/or ReadtheDocs and/or website								|
+| 2 		| Copyright holder on specification			| OpenHW on behalf of contributors (member), who hold the copyright jointly. 		|
+| 3			| License used for publication 				| Same license as used for contributions 											|
+| 4			| Copyright Grant included in specification publication 	 | As per publication license:										}	
+| 5.1		| Apache 2.0								| Yes 																				|
+| 5.2		| Solderpad 2.1								| Yes 																				|
+| 5.3		| Commons Attribution Share Alike 4.0 International license | Yes																|								
+| 6			| Patent Grant included in specification publication 	|	As per publication license											|
+| 6.1		| Apache 2.0								| Yes 																				|
+| 6.2		| Solderpad 2.1								| Yes 																				|
+| 6.3		| Commons Attribution Share Alike 4.0 International license | No 																|								
+| 7			| Essential patents disclosed by OpenHW		| No																				|
+| 8 		| License text included within the specification text |  No - referall only 													|
+| 0			| "No-warranty" or disclaimer				| Required																|
+
+
+### Specification States
+
+The ratification or completion state ais meant to be written in each OpenHW specification so that a reader will be aware of the state of completion of the Specification
+These are the OpenHW Specification States:
+
+- “OpenHW Specification - In Development"
+- "OpenHW Specification - In Review"
+- "OpenHW Specification - Released"
+
+#### "OpenHW Specification - In Development"
+
+When the specification is initiated, all drafts will be labelled as “OpenHW Specification - In Development"
+
+Revision numbers should be of the form 0.Y.Z
```

Comment:
0.Y.Z only works for the first revision.  At some point we will have a Release version, say 1.0.0, and others will wish to update it to (say) 1.1.0, or even 2.0.0.

Recommend the following:
```
Revision numbers shall be of the form X.Y.Z with X=0 for the initial version of the specification.
```
(Also note change of "should" to "shall".)

Reply:
i read this as development versions shall start with 0.
would be good to add a reference to semantic versioning

Reply:
Latest version of the doc has the reference to semantic versioning.  Marking this conversation "resolved".